query/src/configs/config.rs

```diff
@@ -113,33 +113,30 @@ pub struct LogConfig {
 
 impl LogConfig {
     pub fn default() -> Self {
-        LogConfig {
-            log_level: "INFO".to_string(),
-            log_dir: "./_logs".to_string(),
-        }
+        <Self as StructOpt>::from_iter(&Vec::<&'static str>::new())
     }
 }
 
 /// Store config group.
 /// serde(default) make the toml de to default working.
 #[derive(Clone, serde::Serialize, serde::Deserialize, PartialEq, StructOpt, StructOptToml)]
 pub struct StoreConfig {
-    #[structopt(long, env = STORE_ADDRESS, default_value = "", help = "Store backend address")]
+    #[structopt(long, env = STORE_ADDRESS, default_value, help = "Store backend address")]
```

Comment:
The change is due to https://docs.rs/structopt/0.3.23/structopt/index.html#default-values